src/parsers/manifestjson.js

```diff
@@ -314,7 +334,8 @@ export default class ManifestJSONParser extends JSONParser {
       validate = validateDictionary;
     }
 
-    this.isValid = validate(this.parsedJSON);
+    this.isValid = validate(this.parsedJSON, this.validatorOptions);
```

Comment:
Ha, I see now. So `validatorOptions` refers to the schema validator, right? Maybe the name should include "schema" somehow then.

Reply:
I can definitely add `schema` in the name of the property, but to be fair the manifestjson parser is the only thing that does actually use the JSONSchema validator and so in the context of this particular class I do think that `this.validatorOptions` may as well be considered as the set of options passed to the related ajv validator instance and so I didn't bothered adding `scheme` to the name.

If you think that it may make it easier to understand what this property does, I'll rename the property (it should be just a couple of changes and so not really a bit deal to change it).